benchmark/benchmarks.jl

```diff
@@ -4,6 +4,14 @@ const SUITE = BenchmarkGroup(["Agents"])
 
 include("agents.jl")
 
+#### SPACE CONSTRUCTION ####
+
+SUITE["space_creation"] = BenchmarkGroup(["graph", "grid", "continuous"])
+SUITE["space_creation"]["graph"] = @benchmarkable GraphSpace(complete_digraph(1000))
+SUITE["space_creation"]["grid"] = @benchmarkable GridSpace((500, 500))
+SUITE["space_creation"]["continuous"] =
+    @benchmarkable ContinuousSpace(5; extend = (100, 100, 100, 100, 100))
+
```

Comment:
this 1 time cost can be excluded to make CI run faster and also reduce the printed comparison. In general anything that happens only 1 doesn't need to be reported here I Would say.